power.mss

```diff
@@ -1,9 +1,10 @@
-@power-line-color: #888;
+@power-live-color: #888;
+@power-support-color: darken(@power-live-color, 15%);
```

Comment:
This PR intended to distinguish power supports from live elements. Introducing a 15%-darker color than live element was done on purpose.
It can be changed if this doesn't meet requirements for carto-style. If not, having darker supports than lines sounds more accurate to me.

Reply:
I agree that they should be distinguished. Currently this is done by using `#888888` for lines and `#928f8f` for poles  - the later color is lighter, because the rendered icons are larger than the lines and do not need as strong of a color to be clearly visible. But it might be reasonable to change the current power=tower icon color (`#868686`) to match `power=pole` and the new `power=portal` icon.

> having darker supports than lines sounds more accurate to me

The rendering is not based on how the features appear in a satellite image or in real life, but based on what provides a legible, understandable map rendering for general users, so differences in appearance should be based on functional or practical differences between features.

Darkening the color to `#636363` makes power poles too prominent compared to buildings and the power lines:

z17 power=pole and =minor_line before
![z17-power-poles-before](https://user-images.githubusercontent.com/42757252/64745616-8b69c280-d543-11e9-8b2b-86add3d2f566.png)

z17 after
![z17-power-poles-after](https://user-images.githubusercontent.com/42757252/64745635-945a9400-d543-11e9-8c6d-9a85830e3cdb.png)

Reply:
It might be worth making power poles a little darker. As right now they are kind of hard to see. Especially if multiple power lines are coming out of them. Although I agree #636363 is probably to dark.

Reply:
Let's remain on original poles colour.